src/themes/shared/fontSizes.ts

```diff
@@ -1,4 +1,5 @@
 export const fontSizes = {
+  xxs: '0.5rem',
```

Comment:
Did you doble check with the designer if we could use `xs` instead adding a new token? And can you make sure if we really need this new token in future cases? Thank you!

Reply:
@boyarskiy any update here?

Reply:
We do not know if we need this token in the future, but we still want to use that size for the close button icon.

Reply:
In case that we add it in our tokens, Christian will need to update this with the Designers and update the figma file -> https://www.figma.com/file/WmNmJ7jdB0Z7tOjj516nFB/01.-AS24---Tokens?node-id=25%3A222&t=EO8Nqhj6nlvahEFq-0

Would you mid letting him know about it? Thank you   🙏🏽